package.json

```diff
@@ -21,6 +21,7 @@
     "api": "npm start -w packages/api",
     "web": "npm start -w packages/website",
     "test": "npm run lint && npm run test --workspaces",
+    "test:e2e": "npx playwright test",
```

Comment:
Shouldn't this live in website package json?

Reply:
I logged this decision in the PR:
> * Is it better to add @playwright/test in the repo root or in packages/website?
>   * originally I did the latter, but it felt awkward because at test-time it also needed packages/api running. Because the tests span packages, I put them in repo root. I don't feel very strongly about it either way. I think as more package-specific tests/configuration are added, it will be easy to colocate them in their appropriate packages and link to them from the root config

Reply:
I was expecting consistency on the scripts, so I would expect this to be in website, as part of its `test` script, which would be triggered by the `"web": "npm start -w packages/website",` of main package.

However, that is a good point that this is `e2e` testing and uses multiple packages. I also not have a strong preference, so would probably put it in line with how we do in nft.storage for consistency.

Reply:
@vasco-santos I'll try that

Reply:
@vasco-santos I've updated to move all the testing stuff back into packages/website.